dept44-starter/src/test/java/se/sundsvall/dept44/support/DotEnvSupportTest.java

```diff
@@ -0,0 +1,68 @@
+package se.sundsvall.dept44.support;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import se.sundsvall.dept44.support.testutil.TestApplication;
+
+@SpringBootTest(classes = TestApplication.class)
+@ActiveProfiles("junit")
+class DotEnvSupportTest {
+
+	/*****************************************
+	 * From .env file
+	 *****************************************/
+	@Value("${this.is.an.env.key:}")
+	private String envValueSpringStyle;
+
+	@Value("${ENV_KEY_1:}")
+	private String envValue1;
+
+	@Value("${ENV_KEY_2:}")
+	private String envValue2;
+
+	/*****************************************
+	 * From properties file (junit)
+	 *****************************************/
+	@Value("${property.key.spring.style:}")
+	private String propertyValueSpringStyle;
+
+	@Value("${property.key.1:}")
+	private String propertyValue1;
+
+	@Value("${property.key.2:}")
+	private String propertyValue2;
+
+	/*****************************************
+	 * Should be empty (test legacy prefixing)
+	 *****************************************/
+	@Value("${env.ENV_KEY_1:}")
+	private String envValueLegacyPrefixed1;
+
+	@Value("${env.ENV_KEY_2:}")
+	private String envValueLegacyPrefixed2;
+
```

Comment:
Add one property where some value is assigned in application.properties and override is done in .env

Reply:
Fixed!